web/packages/shared/components/UnifiedResources/FilterPanel.tsx

```diff
@@ -494,6 +512,124 @@ function ViewModeSwitch({
   );
 }
 
+const options: { value: IncludedResourceMode; label: string }[] = [
+  {
+    value: 'accessible',
+    label: 'Available',
+  },
+  {
+    value: 'requestable',
+    label: 'Can be requested',
+  },
+];
+
+const IncludedResourcesSelector = ({
+  onChange,
+  availabilityFilter,
+}: {
+  onChange: (value: IncludedResourceMode) => void;
+  availabilityFilter: ResourceAvailabilityFilter;
+}) => {
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  const handleOpen = event => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  function applyFilter(e: FormEvent<HTMLFormElement>) {
+    e.preventDefault();
+    handleClose();
+
+    const formData = new FormData(e.currentTarget);
+    const availabilityOptionsForm = formData.getAll('availabilityOptions');
+
+    // We selected all or nothing.
+    if (
+      availabilityOptionsForm.length === 0 ||
+      availabilityOptionsForm.length === 2
+    ) {
+      onChange('all');
+    } else {
+      onChange(availabilityOptionsForm.at(0) as IncludedResourceMode);
+    }
+  }
+
+  function isCheckboxPreSelected(option: IncludedResourceMode): boolean {
+    return availabilityFilter.mode === option;
+  }
+
+  return (
+    <Flex textAlign="center" alignItems="center">
+      <HoverTooltip tipContent={'Filter by resource availability'}>
+        <ButtonSecondary
+          px={2}
+          css={`
+            border-color: ${props => props.theme.colors.spotBackground[0]};
+          `}
+          textTransform="none"
+          size="small"
+          onClick={handleOpen}
+        >
+          Availability
+          <ChevronDown ml={2} size="small" color="text.slightlyMuted" />
+          {availabilityFilter.canRequestAll === true &&
+            availabilityFilter.mode !== 'all' && <FiltersExistIndicator />}
+        </ButtonSecondary>
+      </HoverTooltip>
+      <Menu
+        popoverCss={() => `
+          margin-top: 36px; 
```

Comment:
I'm not sure if this is the reason, but the menu for availability renders higher than other menus.